schema/v3.5.0.cql

```diff
@@ -0,0 +1 @@
+ALTER TABLE backup_run_progress ADD scylla_task_id text;
```

Comment:
nit: Why new parameter is needed ? Cannot agent_job_id be used ? (it's not a complain, just the curiosity)

Reply:
They have different types and it also allow for distinguishing which API was used.